Many broken benchmarks in test suite

I tried to run the entire suite of benchmarks just now from 4c69794ba41ea154787191d561acb92f4808a974, and I found that many are failing:
```
$ go test -run=^$ -benchtime=1x -bench=. ./...
PASS
ok  	github.com/influxdata/flux	0.658s
PASS
ok  	github.com/influxdata/flux/arrow	0.008s
PASS
ok  	github.com/influxdata/flux/ast	0.013s
?   	github.com/influxdata/flux/ast/asttest	[no test files]
?   	github.com/influxdata/flux/ast/asttest/cmpgen	[no test files]
PASS
ok  	github.com/influxdata/flux/ast/edit	0.012s
PASS
ok  	github.com/influxdata/flux/builtin	0.655s
?   	github.com/influxdata/flux/cmd/flux	[no test files]
?   	github.com/influxdata/flux/cmd/flux/cmd	[no test files]
PASS
ok  	github.com/influxdata/flux/compiler	0.008s
PASS
ok  	github.com/influxdata/flux/complete	0.009s
PASS
ok  	github.com/influxdata/flux/control	0.787s
?   	github.com/influxdata/flux/control/controltest	[no test files]
PASS
ok  	github.com/influxdata/flux/csv	0.020s
PASS
ok  	github.com/influxdata/flux/examples/library	0.692s
goos: darwin
goarch: amd64
pkg: github.com/influxdata/flux/execute
BenchmarkGroupLookup_LookupOrCreate/Ordered-4         	       1	     75689 ns/op	   10648 B/op	      10 allocs/op
BenchmarkGroupLookup_LookupOrCreate/Interweaved-4     	       1	    143010 ns/op	   10456 B/op	      40 allocs/op
BenchmarkGroupLookup_LookupOrCreate/Unordered-4       	       1	    221417 ns/op	   11768 B/op	     147 allocs/op
BenchmarkGroupLookup_LookupOrCreate/Reversed-4        	       1	    317405 ns/op	   12120 B/op	     218 allocs/op
BenchmarkGroupLookup_LookupOrCreate/ManySplits-4      	       1	    496330 ns/op	   12712 B/op	     148 allocs/op
PASS
ok  	github.com/influxdata/flux/execute	0.659s
?   	github.com/influxdata/flux/execute/executetest	[no test files]
PASS
ok  	github.com/influxdata/flux/influxql	0.023s
?   	github.com/influxdata/flux/internal/cmd/builtin	[no test files]
?   	github.com/influxdata/flux/internal/cmd/builtin/cmd	[no test files]
?   	github.com/influxdata/flux/internal/cmd/changelog	[no test files]
?   	github.com/influxdata/flux/internal/cmd/changelog/cmd	[no test files]
?   	github.com/influxdata/flux/internal/cmd/refactortests	[no test files]
PASS
ok  	github.com/influxdata/flux/internal/cmd/refactortests/cmd	0.662s
?   	github.com/influxdata/flux/internal/gen	[no test files]
goos: darwin
goarch: amd64
pkg: github.com/influxdata/flux/internal/parser
BenchmarkParser/package_clause-4         	       1	    167714 ns/op	    9520 B/op	     247 allocs/op
BenchmarkParser/import-4                 	       1	    127727 ns/op	    8240 B/op	     237 allocs/op
BenchmarkParser/import_as-4              	       1	    237662 ns/op	   10720 B/op	     302 allocs/op
BenchmarkParser/imports-4                	       1	    384850 ns/op	   12088 B/op	     361 allocs/op
BenchmarkParser/package_and_imports-4    	       1	    250122 ns/op	   15624 B/op	     480 allocs/op
BenchmarkParser/package_and_imports_and_body-4         	       1	    457322 ns/op	   25784 B/op	     760 allocs/op
BenchmarkParser/optional_query_metadata-4              	       1	    526476 ns/op	   44408 B/op	    1373 allocs/op
BenchmarkParser/optional_query_metadata_preceding_query_text-4         	       1	    481425 ns/op	   36904 B/op	    1147 allocs/op
BenchmarkParser/qualified_option-4                                     	       1	    178738 ns/op	   16576 B/op	     495 allocs/op
BenchmarkParser/builtin-4                                              	       1	    110139 ns/op	    8288 B/op	     237 allocs/op
BenchmarkParser/test-4                                                 	       1	    337313 ns/op	   24328 B/op	     753 allocs/op
BenchmarkParser/from-4                                                 	       1	    246071 ns/op	   11168 B/op	     311 allocs/op
BenchmarkParser/comment-4                                              	       1	    207404 ns/op	   11904 B/op	     318 allocs/op
BenchmarkParser/identifier_with_number-4                               	       1	    327264 ns/op	   11760 B/op	     314 allocs/op
BenchmarkParser/regex_literal-4                                        	       1	    180001 ns/op	    9560 B/op	     264 allocs/op
BenchmarkParser/regex_literal_with_escape_sequence-4                   	       1	    182438 ns/op	   10120 B/op	     273 allocs/op
BenchmarkParser/bad_regex_literal-4                                    	       1	    379065 ns/op	    9424 B/op	     265 allocs/op
BenchmarkParser/regex_match_operators-4                                	       1	    311710 ns/op	   23200 B/op	     688 allocs/op
BenchmarkParser/declare_variable_as_an_int-4                           	       1	    147437 ns/op	   11296 B/op	     318 allocs/op
BenchmarkParser/declare_variable_as_a_float-4                          	       1	    242241 ns/op	   10656 B/op	     314 allocs/op
BenchmarkParser/declare_variable_as_an_array-4                         	       1	    534889 ns/op	   19872 B/op	     602 allocs/op
BenchmarkParser/declare_variable_as_an_empty_array-4                   	       1	    364491 ns/op	   11600 B/op	     322 allocs/op
BenchmarkParser/use_variable_to_declare_something-4                    	       1	    170761 ns/op	   16848 B/op	     499 allocs/op
BenchmarkParser/variable_is_from_statement-4                           	       1	    204490 ns/op	   22576 B/op	     688 allocs/op
BenchmarkParser/pipe_expression-4                                      	       1	    261912 ns/op	   16848 B/op	     493 allocs/op
BenchmarkParser/pipe_expression_to_member_expression_function-4        	       1	    421148 ns/op	   26800 B/op	     814 allocs/op
BenchmarkParser/literal_pipe_expression-4                              	       1	    194304 ns/op	   15520 B/op	     439 allocs/op
BenchmarkParser/member_expression_pipe_expression-4                    	       1	    230435 ns/op	   18560 B/op	     555 allocs/op
BenchmarkParser/multiple_pipe_expressions-4                            	       1	    418312 ns/op	   27968 B/op	     858 allocs/op
BenchmarkParser/pipe_expression_into_non-call_expression-4             	       1	    243806 ns/op	   15360 B/op	     445 allocs/op
BenchmarkParser/two_variables_for_two_froms-4                          	       1	    497441 ns/op	   39712 B/op	    1245 allocs/op
BenchmarkParser/from_with_database-4                                   	       1	    325571 ns/op	   19088 B/op	     572 allocs/op
BenchmarkParser/map_member_expressions-4                               	       1	    642170 ns/op	   38456 B/op	    1206 allocs/op
BenchmarkParser/object_with_string_literal_key-4                       	       1	    302835 ns/op	   16736 B/op	     504 allocs/op
BenchmarkParser/object_with_mixed_keys-4                               	       1	    950606 ns/op	   23096 B/op	     705 allocs/op
BenchmarkParser/implicit_key_object_literal-4                          	       1	    402688 ns/op	   17560 B/op	     491 allocs/op
BenchmarkParser/index_expression-4                                     	       1	    464433 ns/op	   13120 B/op	     376 allocs/op
BenchmarkParser/nested_index_expression-4                              	       1	    285776 ns/op	   17664 B/op	     517 allocs/op
BenchmarkParser/access_indexed_object_returned_from_function_call-4    	       1	    217405 ns/op	   15760 B/op	     451 allocs/op
BenchmarkParser/index_with_member_expressions-4                        	       1	    242795 ns/op	   16880 B/op	     500 allocs/op
BenchmarkParser/index_with_member_with_call_expression-4               	       1	    370920 ns/op	   18896 B/op	     565 allocs/op
BenchmarkParser/index_with_unclosed_bracket-4                          	       1	    165053 ns/op	   15616 B/op	     453 allocs/op
BenchmarkParser/index_with_unbalanced_parenthesis-4                    	       1	    199883 ns/op	   15600 B/op	     453 allocs/op
BenchmarkParser/index_with_unexpected_rparen-4                         	       1	    178944 ns/op	   13872 B/op	     402 allocs/op
BenchmarkParser/binary_expression-4                                    	       1	    238470 ns/op	   12928 B/op	     375 allocs/op
BenchmarkParser/member_expression_binary_expression-4                  	       1	    274819 ns/op	   16768 B/op	     497 allocs/op
BenchmarkParser/var_as_binary_expression_of_other_vars-4               	       1	    488955 ns/op	   32080 B/op	    1000 allocs/op
BenchmarkParser/var_as_unary_expression_of_other_vars-4                	       1	    297425 ns/op	   18528 B/op	     560 allocs/op
BenchmarkParser/var_as_both_binary_and_unary_expressions-4             	       1	    394052 ns/op	   22320 B/op	     686 allocs/op
BenchmarkParser/unary_expressions_within_logical_expression-4          	       1	    718391 ns/op	   34480 B/op	    1083 allocs/op
BenchmarkParser/unary_expressions_with_too_many_comments-4             	       1	    325214 ns/op	   34256 B/op	    1084 allocs/op
BenchmarkParser/expressions_with_function_calls-4                      	       1	    168133 ns/op	   16768 B/op	     501 allocs/op
BenchmarkParser/mix_unary_logical_and_binary_expressions-4             	       1	    362946 ns/op	   28464 B/op	     859 allocs/op
BenchmarkParser/mix_unary_logical_and_binary_expressions_with_extra_parens-4         	       1	    603145 ns/op	   27280 B/op	     848 allocs/op
BenchmarkParser/binary_operator_precedence-4                                         	       1	    182185 ns/op	   16640 B/op	     495 allocs/op
BenchmarkParser/binary_operator_precedence_-_literals_only-4                         	       1	    183132 ns/op	   16704 B/op	     502 allocs/op
BenchmarkParser/binary_operator_precedence_-_double_subtraction-4                    	       1	    215649 ns/op	   17440 B/op	     510 allocs/op
BenchmarkParser/binary_operator_precedence_-_double_subtraction_with_parens-4        	       1	    169386 ns/op	   17168 B/op	     512 allocs/op
BenchmarkParser/binary_operator_precedence_-_double_sum-4                            	       1	    163614 ns/op	   16736 B/op	     499 allocs/op
BenchmarkParser/binary_operator_precedence_-_double_sum_with_parens-4                	       1	    171650 ns/op	   17168 B/op	     512 allocs/op
BenchmarkParser/logical_unary_operator_precedence-4                                  	       1	    165266 ns/op	   16688 B/op	     500 allocs/op
BenchmarkParser/all_operators_precedence-4                                           	       1	    803912 ns/op	  116400 B/op	    3684 allocs/op
BenchmarkParser/logical_operators_precedence_1-4                                     	       1	    164896 ns/op	   15024 B/op	     446 allocs/op
BenchmarkParser/logical_operators_precedence_2-4                                     	       1	    223450 ns/op	   14976 B/op	     442 allocs/op
BenchmarkParser/logical_operators_precedence_3-4                                     	       1	    235115 ns/op	   15072 B/op	     445 allocs/op
BenchmarkParser/logical_operators_precedence_4-4                                     	       1	    246645 ns/op	   14928 B/op	     440 allocs/op
BenchmarkParser/logical_operators_precedence_5-4                                     	       1	    242923 ns/op	   16928 B/op	     507 allocs/op
BenchmarkParser/logical_operators_precedence_6-4                                     	       1	    252805 ns/op	   16928 B/op	     507 allocs/op
BenchmarkParser/logical_operators_precedence_7-4                                     	       1	    328345 ns/op	   15424 B/op	     455 allocs/op
BenchmarkParser/logical_operators_precedence_8-4                                     	       1	    234313 ns/op	   15376 B/op	     453 allocs/op
BenchmarkParser/logical_operators_precedence_9-4                                     	       1	    178779 ns/op	   17360 B/op	     520 allocs/op
BenchmarkParser/logical_operators_precedence_10-4                                    	       1	    220901 ns/op	   17312 B/op	     516 allocs/op
BenchmarkParser/two_logical_operations_with_parens-4                                 	       1	    383239 ns/op	   28248 B/op	     860 allocs/op
BenchmarkParser/arrow_function_called-4                                              	       1	    694270 ns/op	   34304 B/op	    1074 allocs/op
BenchmarkParser/arrow_function_return_map-4                                          	       1	    260644 ns/op	   22960 B/op	     711 allocs/op
BenchmarkParser/arrow_function_with_default_arg-4                                    	       1	    239863 ns/op	   26512 B/op	     823 allocs/op
BenchmarkParser/arrow_function_called_in_binary_expression-4                         	       1	    350398 ns/op	   44080 B/op	    1398 allocs/op
BenchmarkParser/arrow_function_as_single_expression-4                                	       1	    251714 ns/op	   24848 B/op	     768 allocs/op
BenchmarkParser/arrow_function_as_block-4                                            	       1	    333404 ns/op	   34592 B/op	    1079 allocs/op
BenchmarkParser/conditional-4                                                        	       1	    192249 ns/op	   17040 B/op	     515 allocs/op
BenchmarkParser/nested_conditionals-4                                                	       1	    404020 ns/op	   45536 B/op	    1469 allocs/op
BenchmarkParser/from_with_filter_with_no_parens-4                                    	       1	    591989 ns/op	   70992 B/op	    2247 allocs/op
BenchmarkParser/from_with_range-4                                                    	       1	    479190 ns/op	   40600 B/op	    1277 allocs/op
BenchmarkParser/from_with_limit-4                                                    	       1	    548234 ns/op	   38296 B/op	    1204 allocs/op
BenchmarkParser/from_with_range_and_count-4                                          	       1	    651383 ns/op	   49192 B/op	    1540 allocs/op
BenchmarkParser/from_with_range,_limit_and_count-4                                   	       1	    909584 ns/op	   63960 B/op	    1974 allocs/op
BenchmarkParser/from_with_join-4                                                     	       1	    848847 ns/op	  120024 B/op	    3868 allocs/op
BenchmarkParser/from_with_join_with_complex_expression-4                             	       1	   1122995 ns/op	  180904 B/op	    5867 allocs/op
BenchmarkParser/duration_literal,_all_units-4                                        	       1	    340830 ns/op	   14064 B/op	     370 allocs/op
BenchmarkParser/duration_literal,_months-4                                           	       1	    244382 ns/op	   10800 B/op	     321 allocs/op
BenchmarkParser/duration_literal,_milliseconds-4                                     	       1	    139040 ns/op	   10816 B/op	     321 allocs/op
BenchmarkParser/duration_literal,_months,_minutes,_milliseconds-4                    	       1	    220199 ns/op	   11936 B/op	     334 allocs/op
BenchmarkParser/date_literal_in_the_default_location-4                               	       1	    180295 ns/op	   10760 B/op	     319 allocs/op
BenchmarkParser/date_time_literal-4                                                  	       1	    213409 ns/op	   10840 B/op	     319 allocs/op
BenchmarkParser/date_time_literal_with_fractional_seconds-4                          	       1	    169491 ns/op	   10872 B/op	     318 allocs/op
BenchmarkParser/unary_expression_with_member_expression-4                            	       1	    290355 ns/op	   14912 B/op	     435 allocs/op
BenchmarkParser/function_call_with_unbalanced_braces-4                               	       1	    596361 ns/op	   46864 B/op	    1412 allocs/op
BenchmarkParser/string_with_utf-8-4                                                  	       1	    230432 ns/op	    9232 B/op	     253 allocs/op
BenchmarkParser/string_with_byte_values-4                                            	       1	    210795 ns/op	    8768 B/op	     249 allocs/op
BenchmarkParser/string_with_escapes-4                                                	       1	    360726 ns/op	    9248 B/op	     254 allocs/op
BenchmarkParser/multiline_string-4                                                   	       1	    231131 ns/op	    9248 B/op	     259 allocs/op
BenchmarkParser/illegal_statement_token-4                                            	       1	    230486 ns/op	   10880 B/op	     320 allocs/op
BenchmarkParser/multiple_idents_in_parens-4                                          	       1	    715099 ns/op	   14128 B/op	     398 allocs/op
BenchmarkParser/missing_left_hand_side-4                                             	       1	    160615 ns/op	    8768 B/op	     212 allocs/op
BenchmarkParser/missing_right_hand_side-4                                            	       1	    154659 ns/op	    9040 B/op	     213 allocs/op
BenchmarkParser/illegal_expression-4                                                 	       1	    309893 ns/op	    4992 B/op	     114 allocs/op
BenchmarkParser/missing_arrow_in_function_expression-4                               	       1	    308813 ns/op	   16048 B/op	     443 allocs/op
BenchmarkParser/property_list_missing_property-4                                     	       1	    352559 ns/op	   24856 B/op	     761 allocs/op
BenchmarkParser/property_list_missing_key-4                                          	       1	    224648 ns/op	   15440 B/op	     457 allocs/op
BenchmarkParser/property_list_missing_value-4                                        	       1	    168135 ns/op	   13936 B/op	     397 allocs/op
BenchmarkParser/property_list_missing_comma-4                                        	       1	    233846 ns/op	   25800 B/op	     796 allocs/op
BenchmarkParser/property_list_trailing_comma-4                                       	       1	    173421 ns/op	   16720 B/op	     502 allocs/op
BenchmarkParser/property_list_bad_property-4                                         	       1	    219942 ns/op	   25192 B/op	     779 allocs/op
BenchmarkParser/invalid_expression_in_array-4                                        	       1	    153397 ns/op	   11520 B/op	     327 allocs/op
PASS
ok  	github.com/influxdata/flux/internal/parser	0.227s
?   	github.com/influxdata/flux/internal/pkg/syncutil	[no test files]
PASS
ok  	github.com/influxdata/flux/internal/scanner	0.010s
?   	github.com/influxdata/flux/internal/spec	[no test files]
PASS
ok  	github.com/influxdata/flux/internal/token	0.010s
PASS
ok  	github.com/influxdata/flux/interpreter	0.009s
?   	github.com/influxdata/flux/interpreter/interptest	[no test files]
?   	github.com/influxdata/flux/iocounter	[no test files]
PASS
ok  	github.com/influxdata/flux/lang	0.680s
PASS
ok  	github.com/influxdata/flux/line	0.019s
PASS
ok  	github.com/influxdata/flux/memory	0.011s
?   	github.com/influxdata/flux/mock	[no test files]
PASS
ok  	github.com/influxdata/flux/parser	0.012s
PASS
ok  	github.com/influxdata/flux/plan	0.706s
?   	github.com/influxdata/flux/plan/plantest	[no test files]
?   	github.com/influxdata/flux/querytest	[no test files]
?   	github.com/influxdata/flux/repl	[no test files]
PASS
ok  	github.com/influxdata/flux/semantic	0.010s
?   	github.com/influxdata/flux/semantic/semantictest	[no test files]
goos: darwin
goarch: amd64
pkg: github.com/influxdata/flux/stdlib
BenchmarkFluxEndToEnd/aggregate_empty_window.flux-4         	       1	  33577080 ns/op	16537824 B/op	  217374 allocs/op
BenchmarkFluxEndToEnd/aggregate_fill_window.flux-4          	       1	  38344204 ns/op	18665008 B/op	  261977 allocs/op
BenchmarkFluxEndToEnd/aggregate_window.flux-4               	       1	  39247443 ns/op	16813312 B/op	  221644 allocs/op
BenchmarkFluxEndToEnd/aggregate_window_max.flux-4           	       1	  35382964 ns/op	16741160 B/op	  221031 allocs/op
BenchmarkFluxEndToEnd/columns.flux-4                        	       1	  34347222 ns/op	13929496 B/op	  168381 allocs/op
BenchmarkFluxEndToEnd/count.flux-4                          	       1	  30972899 ns/op	13970648 B/op	  170731 allocs/op
BenchmarkFluxEndToEnd/cov.flux-4                            	--- FAIL: BenchmarkFluxEndToEnd/cov.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield13
BenchmarkFluxEndToEnd/covariance.flux-4                     	       1	  40794589 ns/op	14172288 B/op	  175745 allocs/op
BenchmarkFluxEndToEnd/cumulative_sum.flux-4                 	       1	  58636549 ns/op	14568776 B/op	  179910 allocs/op
BenchmarkFluxEndToEnd/cumulative_sum_default.flux-4         	       1	  31853682 ns/op	13846248 B/op	  169506 allocs/op
BenchmarkFluxEndToEnd/cumulative_sum_noop.flux-4            	       1	  34286804 ns/op	13713744 B/op	  167568 allocs/op
BenchmarkFluxEndToEnd/derivative.flux-4                     	       1	  32780718 ns/op	13902416 B/op	  171045 allocs/op
BenchmarkFluxEndToEnd/derivative_nonnegative.flux-4         	       1	  30562654 ns/op	14242064 B/op	  176442 allocs/op
BenchmarkFluxEndToEnd/difference.flux-4                     	       1	  35020973 ns/op	13856456 B/op	  168813 allocs/op
BenchmarkFluxEndToEnd/difference_columns.flux-4             	       1	  34082971 ns/op	14463688 B/op	  179015 allocs/op
BenchmarkFluxEndToEnd/difference_nonnegative.flux-4         	       1	  43101949 ns/op	14122568 B/op	  173207 allocs/op
BenchmarkFluxEndToEnd/difference_one_value.flux-4           	       1	  34610118 ns/op	14011160 B/op	  171667 allocs/op
BenchmarkFluxEndToEnd/difference_panic.flux-4               	--- FAIL: BenchmarkFluxEndToEnd/difference_panic.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/distinct.flux-4                       	       1	  33925665 ns/op	15179184 B/op	  188831 allocs/op
BenchmarkFluxEndToEnd/drop_after_rename.flux-4              	       1	  35543413 ns/op	16047072 B/op	  201849 allocs/op
BenchmarkFluxEndToEnd/drop_fn.flux-4                        	--- FAIL: BenchmarkFluxEndToEnd/drop_fn.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield5
BenchmarkFluxEndToEnd/drop_newname_after.flux-4             	       1	  33010118 ns/op	15929656 B/op	  200492 allocs/op
BenchmarkFluxEndToEnd/drop_newname_before.flux-4            	       1	  37621824 ns/op	15809472 B/op	  200044 allocs/op
BenchmarkFluxEndToEnd/drop_unused.flux-4                    	--- FAIL: BenchmarkFluxEndToEnd/drop_unused.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/duplicate.flux-4                      	       1	  33575401 ns/op	14136168 B/op	  174682 allocs/op
BenchmarkFluxEndToEnd/duplicate_overwrite.flux-4            	       1	  32739816 ns/op	14284768 B/op	  176590 allocs/op
BenchmarkFluxEndToEnd/fill_bool.flux-4                      	       1	  32162943 ns/op	14131800 B/op	  173255 allocs/op
BenchmarkFluxEndToEnd/fill_float.flux-4                     	       1	  32360982 ns/op	14171664 B/op	  174309 allocs/op
BenchmarkFluxEndToEnd/fill_int.flux-4                       	       1	  34739212 ns/op	14408072 B/op	  178528 allocs/op
BenchmarkFluxEndToEnd/fill_previous.flux-4                  	       1	  31230955 ns/op	12592408 B/op	  155751 allocs/op
BenchmarkFluxEndToEnd/fill_string.flux-4                    	       1	  31196293 ns/op	14230848 B/op	  175848 allocs/op
BenchmarkFluxEndToEnd/fill_time.flux-4                      	       1	  31358405 ns/op	13640472 B/op	  165081 allocs/op
BenchmarkFluxEndToEnd/fill_uint.flux-4                      	       1	  35367474 ns/op	15027824 B/op	  187128 allocs/op
BenchmarkFluxEndToEnd/filter_by_regex.flux-4                	--- FAIL: BenchmarkFluxEndToEnd/filter_by_regex.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield5
BenchmarkFluxEndToEnd/filter_by_regex_function.flux-4       	--- FAIL: BenchmarkFluxEndToEnd/filter_by_regex_function.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/filter_by_tags.flux-4                 	--- FAIL: BenchmarkFluxEndToEnd/filter_by_tags.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield5
BenchmarkFluxEndToEnd/first.flux-4                          	       1	  29214842 ns/op	12767544 B/op	  153805 allocs/op
BenchmarkFluxEndToEnd/group.flux-4                          	--- FAIL: BenchmarkFluxEndToEnd/group.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield6
BenchmarkFluxEndToEnd/group_by_field.flux-4                 	       1	  31773590 ns/op	14369624 B/op	  177472 allocs/op
BenchmarkFluxEndToEnd/group_by_irregular.flux-4             	--- FAIL: BenchmarkFluxEndToEnd/group_by_irregular.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "r1" found on plan node: yield5
BenchmarkFluxEndToEnd/group_except.flux-4                   	       1	  36702585 ns/op	15712072 B/op	  197240 allocs/op
BenchmarkFluxEndToEnd/group_nulls.flux-4                    	       1	  30443229 ns/op	13269424 B/op	  161228 allocs/op
BenchmarkFluxEndToEnd/group_ungroup.flux-4                  	--- FAIL: BenchmarkFluxEndToEnd/group_ungroup.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield5
BenchmarkFluxEndToEnd/highestAverage.flux-4                 	       1	  32840398 ns/op	14344712 B/op	  192401 allocs/op
BenchmarkFluxEndToEnd/highestCurrent.flux-4                 	       1	  40127940 ns/op	16990656 B/op	  242646 allocs/op
BenchmarkFluxEndToEnd/highestMax.flux-4                     	       1	  42864783 ns/op	14364864 B/op	  192927 allocs/op
BenchmarkFluxEndToEnd/histogram.flux-4                      	       1	  32542279 ns/op	13708624 B/op	  167130 allocs/op
BenchmarkFluxEndToEnd/histogram_normalize.flux-4            	       1	  40239057 ns/op	14424208 B/op	  178982 allocs/op
BenchmarkFluxEndToEnd/histogram_quantile.flux-4             	       1	  32385571 ns/op	14719544 B/op	  183169 allocs/op
BenchmarkFluxEndToEnd/histogram_quantile_minvalue.flux-4    	       1	  36890719 ns/op	14428944 B/op	  177887 allocs/op
BenchmarkFluxEndToEnd/increase.flux-4                       	       1	  33380022 ns/op	14520792 B/op	  180919 allocs/op
BenchmarkFluxEndToEnd/influxFieldsAsCols.flux-4             	       1	  44798464 ns/op	17246984 B/op	  215349 allocs/op
BenchmarkFluxEndToEnd/influxRawQuery.flux-4                 	--- FAIL: BenchmarkFluxEndToEnd/influxRawQuery.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield8
BenchmarkFluxEndToEnd/integral.flux-4                       	       1	  38951140 ns/op	12902824 B/op	  156246 allocs/op
BenchmarkFluxEndToEnd/join.flux-4                           	--- FAIL: BenchmarkFluxEndToEnd/join.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield12
BenchmarkFluxEndToEnd/join_across_measurements.flux-4       	--- FAIL: BenchmarkFluxEndToEnd/join_across_measurements.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield8
BenchmarkFluxEndToEnd/keep.flux-4                           	       1	  33824379 ns/op	14550280 B/op	  180018 allocs/op
BenchmarkFluxEndToEnd/keep_fn.flux-4                        	--- FAIL: BenchmarkFluxEndToEnd/keep_fn.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/key_values.flux-4                     	       1	  32750993 ns/op	13077968 B/op	  158767 allocs/op
BenchmarkFluxEndToEnd/key_values_host_name.flux-4           	       1	  33525227 ns/op	13145880 B/op	  160531 allocs/op
BenchmarkFluxEndToEnd/keys.flux-4                           	       1	  33718872 ns/op	13651432 B/op	  167555 allocs/op
BenchmarkFluxEndToEnd/last.flux-4                           	       1	  33860887 ns/op	12746248 B/op	  153817 allocs/op
BenchmarkFluxEndToEnd/limit.flux-4                          	       1	  36785547 ns/op	14566584 B/op	  178678 allocs/op
BenchmarkFluxEndToEnd/limit_offset.flux-4                   	       1	  42253762 ns/op	14733824 B/op	  182355 allocs/op
BenchmarkFluxEndToEnd/lowestAverage.flux-4                  	       1	  39885859 ns/op	16781488 B/op	  240938 allocs/op
BenchmarkFluxEndToEnd/lowestCurrent.flux-4                  	       1	  38769453 ns/op	16948256 B/op	  241948 allocs/op
BenchmarkFluxEndToEnd/lowestMin.flux-4                      	       1	  48747454 ns/op	16793864 B/op	  241069 allocs/op
BenchmarkFluxEndToEnd/map.flux-4                            	--- FAIL: BenchmarkFluxEndToEnd/map.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield4
BenchmarkFluxEndToEnd/map_with_obj.flux-4                   	--- FAIL: BenchmarkFluxEndToEnd/map_with_obj.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield5
BenchmarkFluxEndToEnd/max.flux-4                            	       1	  30037863 ns/op	12729168 B/op	  153591 allocs/op
BenchmarkFluxEndToEnd/mean.flux-4                           	       1	  35667396 ns/op	13650248 B/op	  168686 allocs/op
BenchmarkFluxEndToEnd/meta_query_fields.flux-4              	--- FAIL: BenchmarkFluxEndToEnd/meta_query_fields.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield8
BenchmarkFluxEndToEnd/meta_query_keys.flux-4                	--- FAIL: BenchmarkFluxEndToEnd/meta_query_keys.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/meta_query_measurements.flux-4        	       1	  37676434 ns/op	16404504 B/op	  210686 allocs/op
BenchmarkFluxEndToEnd/min.flux-4                            	       1	  29903081 ns/op	12681376 B/op	  152895 allocs/op
BenchmarkFluxEndToEnd/multiple_range.flux-4                 	       1	  34335574 ns/op	14587280 B/op	  180608 allocs/op
BenchmarkFluxEndToEnd/parse_regex.flux-4                    	--- FAIL: BenchmarkFluxEndToEnd/parse_regex.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/pivot.flux-4                          	       1	  45410775 ns/op	16395840 B/op	  207307 allocs/op
BenchmarkFluxEndToEnd/pivot_col_order.flux-4                	       1	  38123176 ns/op	16437112 B/op	  207557 allocs/op
BenchmarkFluxEndToEnd/pivot_fields.flux-4                   	       1	  37068316 ns/op	16345392 B/op	  206564 allocs/op
BenchmarkFluxEndToEnd/pivot_mean.flux-4                     	       1	  55650390 ns/op	19411960 B/op	  262312 allocs/op
BenchmarkFluxEndToEnd/pivot_task_test.flux-4                	--- FAIL: BenchmarkFluxEndToEnd/pivot_task_test.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/quantile.flux-4                       	       1	  38443431 ns/op	14685288 B/op	  182261 allocs/op
BenchmarkFluxEndToEnd/quantile_aggregate.flux-4             	       1	  35325025 ns/op	14049544 B/op	  173609 allocs/op
BenchmarkFluxEndToEnd/quantile_tdigest.flux-4               	       1	  31655488 ns/op	14356264 B/op	  174898 allocs/op
BenchmarkFluxEndToEnd/range.flux-4                          	       1	  34994835 ns/op	12973336 B/op	  158296 allocs/op
BenchmarkFluxEndToEnd/reduce.flux-4                         	--- FAIL: BenchmarkFluxEndToEnd/reduce.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield5
BenchmarkFluxEndToEnd/rename.flux-4                         	       1	  39169687 ns/op	15943664 B/op	  201634 allocs/op
BenchmarkFluxEndToEnd/rename_fn.flux-4                      	--- FAIL: BenchmarkFluxEndToEnd/rename_fn.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield6
BenchmarkFluxEndToEnd/rename_multiple.flux-4                	       1	  38615394 ns/op	16016864 B/op	  201219 allocs/op
BenchmarkFluxEndToEnd/sample.flux-4                         	       1	  36071314 ns/op	13270832 B/op	  160408 allocs/op
BenchmarkFluxEndToEnd/select_measurement.flux-4             	--- FAIL: BenchmarkFluxEndToEnd/select_measurement.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield5
BenchmarkFluxEndToEnd/select_measurement_field.flux-4       	--- FAIL: BenchmarkFluxEndToEnd/select_measurement_field.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield5
BenchmarkFluxEndToEnd/selector_preserve_time.flux-4         	       1	  37745259 ns/op	16228376 B/op	  207066 allocs/op
BenchmarkFluxEndToEnd/set.flux-4                            	       1	  38744489 ns/op	14532008 B/op	  180050 allocs/op
BenchmarkFluxEndToEnd/set_new_column.flux-4                 	       1	  32574923 ns/op	14662472 B/op	  181629 allocs/op
BenchmarkFluxEndToEnd/shift.flux-4                          	       1	  35890232 ns/op	12950952 B/op	  156508 allocs/op
BenchmarkFluxEndToEnd/shift_negative_duration.flux-4        	       1	  31737268 ns/op	13060768 B/op	  157504 allocs/op
BenchmarkFluxEndToEnd/show_all_tag_keys.flux-4              	--- FAIL: BenchmarkFluxEndToEnd/show_all_tag_keys.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield8
BenchmarkFluxEndToEnd/simple_max.flux-4                     	--- FAIL: BenchmarkFluxEndToEnd/simple_max.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield7
BenchmarkFluxEndToEnd/skew.flux-4                           	       1	  33969945 ns/op	13822960 B/op	  168756 allocs/op
BenchmarkFluxEndToEnd/sort.flux-4                           	       1	  39361032 ns/op	14681848 B/op	  180831 allocs/op
BenchmarkFluxEndToEnd/spread.flux-4                         	       1	  35600222 ns/op	13791248 B/op	  167952 allocs/op
BenchmarkFluxEndToEnd/state_count.flux-4                    	--- FAIL: BenchmarkFluxEndToEnd/state_count.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield5
BenchmarkFluxEndToEnd/state_duration.flux-4                 	--- FAIL: BenchmarkFluxEndToEnd/state_duration.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield5
BenchmarkFluxEndToEnd/stddev.flux-4                         	       1	  49311946 ns/op	13689240 B/op	  167083 allocs/op
BenchmarkFluxEndToEnd/string_sort.flux-4                    	       1	  51983970 ns/op	13735600 B/op	  169221 allocs/op
BenchmarkFluxEndToEnd/sum.flux-4                            	       1	  48043060 ns/op	13813912 B/op	  169003 allocs/op
BenchmarkFluxEndToEnd/top.flux-4                            	       1	  54984252 ns/op	14196536 B/op	  177242 allocs/op
BenchmarkFluxEndToEnd/totime.flux-4                         	--- FAIL: BenchmarkFluxEndToEnd/totime.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield5
BenchmarkFluxEndToEnd/union.flux-4                          	--- FAIL: BenchmarkFluxEndToEnd/union.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield11
BenchmarkFluxEndToEnd/union_heterogeneous.flux-4            	--- FAIL: BenchmarkFluxEndToEnd/union_heterogeneous.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "got" found on plan node: yield13
BenchmarkFluxEndToEnd/unique.flux-4                         	       1	  33642082 ns/op	13004232 B/op	  156519 allocs/op
BenchmarkFluxEndToEnd/window.flux-4                         	--- FAIL: BenchmarkFluxEndToEnd/window.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield8
BenchmarkFluxEndToEnd/window_default_start_align.flux-4     	       1	  36278696 ns/op	14431920 B/op	  178609 allocs/op
BenchmarkFluxEndToEnd/window_generate_empty.flux-4          	       1	  33461209 ns/op	14970584 B/op	  186465 allocs/op
BenchmarkFluxEndToEnd/window_group_mean_ungroup.flux-4      	       1	  45983245 ns/op	18344072 B/op	  243490 allocs/op
BenchmarkFluxEndToEnd/window_null.flux-4                    	       1	  34490580 ns/op	14233656 B/op	  174914 allocs/op
BenchmarkFluxEndToEnd/window_offset.flux-4                  	--- FAIL: BenchmarkFluxEndToEnd/window_offset.flux-4
    flux_test.go:114: unexpected error retrieving testing.run result: error in building plan while starting program: duplicate yield name "0" found on plan node: yield8
BenchmarkFluxEndToEnd/window_start_bound.flux-4             	       1	  43802811 ns/op	14354760 B/op	  178000 allocs/op
--- FAIL: BenchmarkFluxEndToEnd
FAIL
exit status 1
FAIL	github.com/influxdata/flux/stdlib	10.564s
PASS
ok  	github.com/influxdata/flux/stdlib/csv	0.740s
?   	github.com/influxdata/flux/stdlib/generate	[no test files]
PASS
ok  	github.com/influxdata/flux/stdlib/http	0.696s
PASS
ok  	github.com/influxdata/flux/stdlib/influxdata/influxdb	0.662s
PASS
ok  	github.com/influxdata/flux/stdlib/influxdata/influxdb/v1	0.661s
PASS
ok  	github.com/influxdata/flux/stdlib/kafka	0.664s
PASS
ok  	github.com/influxdata/flux/stdlib/math	0.009s
PASS
ok  	github.com/influxdata/flux/stdlib/socket	0.646s
?   	github.com/influxdata/flux/stdlib/sql	[no test files]
PASS
ok  	github.com/influxdata/flux/stdlib/strings	0.008s
?   	github.com/influxdata/flux/stdlib/system	[no test files]
PASS
ok  	github.com/influxdata/flux/stdlib/testing	0.023s
goos: darwin
goarch: amd64
pkg: github.com/influxdata/flux/stdlib/universe
BenchmarkCount-4      	       1	      5321 ns/op
BenchmarkFirst-4      	       1	      2590 ns/op
BenchmarkLast-4       	       1	      4429 ns/op
BenchmarkMax-4        	       1	   4609204 ns/op
--- FAIL: BenchmarkMean
    mean_test.go:97: unexpected value -want/+got
        {float64}:
        	-: 10.00081696729983
        	+: 9.99847267384332
BenchmarkMin-4        	       1	   5882184 ns/op
--- FAIL: BenchmarkQuantile
    quantile_test.go:749: unexpected value -want/+got
        {float64}:
        	-: 13.843815760607427
        	+: 13.842132136909889
BenchmarkSample-4     	       1	   1146178 ns/op
--- FAIL: BenchmarkSkew
    skew_test.go:115: unexpected value -want/+got
        {float64}:
        	-: 0.0032200673020400935
        	+: -0.0019606823191321435
--- FAIL: BenchmarkSpread
    spread_test.go:93: unexpected value -want/+got
        {float64}:
        	-: 28.227196461851847
        	+: 31.463516750575685
--- FAIL: BenchmarkStddev
    stddev_test.go:121: unexpected value -want/+got
        {float64}:
        	-: 2.998926113076968
        	+: 3.0024593913384368
--- FAIL: BenchmarkSum
    sum_test.go:96: unexpected value -want/+got
        {float64}:
        	-: 1.000081696729983e+07
        	+: 9.99847267384332e+06
FAIL
exit status 1
FAIL	github.com/influxdata/flux/stdlib/universe	1.726s
PASS
ok  	github.com/influxdata/flux/values	0.008s
```

--- a/execute/executetest/selector.go
+++ b/execute/executetest/selector.go
@@ -81,7 +81,7 @@ func IndexSelectorFuncTestHelper(t *testing.T, selector execute.IndexSelector, d
 	}
 }
 
-func IndexSelectorFuncBenchmarkHelper(b *testing.B, selector execute.IndexSelector, data flux.Table) {
+func IndexSelectorFuncBenchmarkHelper(b *testing.B, selector execute.IndexSelector, data flux.BufferedTable) {
 	b.Helper()
 
 	valueIdx := execute.ColIdx(execute.DefaultValueColLabel, data.Cols())
@@ -93,7 +93,9 @@ func IndexSelectorFuncBenchmarkHelper(b *testing.B, selector execute.IndexSelect
 	var got [][]int
 	for n := 0; n < b.N; n++ {
 		s := selector.NewFloatSelector()
-		if err := data.Do(func(cr flux.ColReader) error {
+
+		t := data.Copy()
+		if err := t.Do(func(cr flux.ColReader) error {
 			got = append(got, s.DoFloat(cr.Floats(valueIdx)))
 			return nil
 		}); err != nil {

--- a/stdlib/flux_test.go
+++ b/stdlib/flux_test.go
@@ -65,10 +65,9 @@ func runEndToEnd(t *testing.T, pkgs []*ast.Package) {
 
 func benchEndToEnd(b *testing.B, pkgs []*ast.Package) {
 	for _, pkg := range pkgs {
-		name := pkg.Files[0].Name
+		name := strings.TrimSuffix(pkg.Files[0].Name, "_test.flux")
 		b.Run(name, func(b *testing.B) {
-			n := strings.TrimSuffix(name, ".flux")
-			if reason, ok := skip[n]; ok {
+			if reason, ok := skip[name]; ok {
 				b.Skip(reason)
 			}
 			b.ResetTimer()

--- a/stdlib/universe/data_test.go
+++ b/stdlib/universe/data_test.go
@@ -24,7 +24,7 @@ const (
 var NormalData []float64
 
 // NormalTable is a table of data whose value col is NormalData.
-var NormalTable flux.Table
+var NormalTable flux.BufferedTable
 
 func init() {
 	dist := distuv.Normal{
@@ -117,5 +117,6 @@ func init() {
 	_ = normalTableBuilder.AppendStrings(4, t1Arrow)
 	_ = normalTableBuilder.AppendStrings(5, t2Arrow)
 
-	NormalTable, _ = normalTableBuilder.Table()
+	table, _ := normalTableBuilder.Table()
+	NormalTable = executetest.MustCopyTable(table).(flux.BufferedTable)
 }

--- a/stdlib/universe/mean_test.go
+++ b/stdlib/universe/mean_test.go
@@ -98,6 +98,6 @@ func BenchmarkMean(b *testing.B) {
 		b,
 		new(universe.MeanAgg),
 		data,
-		10.00081696729983,
+		9.99847267384332,
 	)
 }

--- a/stdlib/universe/quantile_test.go
+++ b/stdlib/universe/quantile_test.go
@@ -753,6 +753,6 @@ func BenchmarkQuantile(b *testing.B) {
 			Compression: 1000,
 		},
 		data,
-		13.843815760607427,
+		13.842132136909889,
 	)
 }

--- a/stdlib/universe/skew_test.go
+++ b/stdlib/universe/skew_test.go
@@ -116,6 +116,6 @@ func BenchmarkSkew(b *testing.B) {
 		b,
 		new(universe.SkewAgg),
 		data,
-		0.0032200673020400935,
+		-0.0019606823191321435,
 	)
 }

--- a/stdlib/universe/spread_test.go
+++ b/stdlib/universe/spread_test.go
@@ -94,6 +94,6 @@ func BenchmarkSpread(b *testing.B) {
 		b,
 		new(universe.SpreadAgg),
 		data,
-		28.227196461851847,
+		31.463516750575685,
 	)
 }

--- a/stdlib/universe/stddev_test.go
+++ b/stdlib/universe/stddev_test.go
@@ -122,6 +122,6 @@ func BenchmarkStddev(b *testing.B) {
 		b,
 		&universe.StddevAgg{Mode: "sample"},
 		data,
-		2.998926113076968,
+		3.0024593913384368,
 	)
 }

--- a/stdlib/universe/sum_test.go
+++ b/stdlib/universe/sum_test.go
@@ -97,6 +97,6 @@ func BenchmarkSum(b *testing.B) {
 		b,
 		new(universe.SumAgg),
 		data,
-		10000816.96729983,
+		9998472.67384332,
 	)
 }

